peppy/project.py

```diff
@@ -1222,6 +1245,27 @@ def _read_sample_data(self):
         :param List[str] sample_table: a list of paths to sample tables
         """
 
+        def _read_tab(pth):
+            """
+            Internal read table function
+
+            :param str pth: absolute path to the file to read
+            :return pandas.DataFrame: table object
+            """
+            csv_kwargs = {
+                "dtype": str,
+                "index_col": False,
+                "keep_default_na": False,
+                "na_values": [""],
+            }
+            try:
+                return pd.read_csv(pth, sep=infer_delimiter(pth), **csv_kwargs)
+            except Exception as e:
+                raise SampleTableFileException(
+                    f"Could not read table: {pth}. "
+                    f"Caught exception: {getattr(e, 'message', repr(e))}"
```

Comment:
Is this a typo? Why 3  arguments to `getattr`?

Reply:
Oh. Is it a default value if the attribute doesn't exist?

Reply:
This were changes proposed by linter so I haven't seen that

Reply:
But yes, it is a default value